services/github/github-release.service.js

```diff
@@ -39,22 +92,106 @@ module.exports = class GithubRelease extends GithubAuthService {
     }
   }
 
-  static render({ version, isPrerelease }) {
-    return {
-      message: addv(version),
-      color: isPrerelease ? 'orange' : 'blue',
+  static render({ version, sort, isPrerelease }) {
+    let color = 'blue'
+    color = sort === 'semver' ? versionColor(version) : color
+    color = isPrerelease ? 'orange' : color
+    return { label: 'release', message: addv(version), color }
+  }
+
+  async fetchReleases({ user, repo }) {
+    return this._requestJson({
+      url: `/repos/${user}/${repo}/releases`,
+      schema: releaseInfoArraySchema,
+      errorMessages: errorMessagesFor('no releases or repo not found'),
```

Comment:
For `/releases` I think we get an http 200 with an empty array correct? IIRC we only get the 404 on `/releases/latest` if there's no releases 

```suggestion
      errorMessages: errorMessagesFor('repo not found'),
```